lib/index.js

```diff
@@ -304,7 +355,7 @@ export default class HyperList {
     }
 
     if (config.applyPatch) {
-      return config.applyPatch(element, fragment)
+      return config.applyPatch(element, fragment, scroller)
     }
 
     element.innerHTML = ''
```

Comment:
There are, like React Faux DOM, but this performs significantly better and allows for a cleaner wrapper to built in React. We might want to approach this as just being VDOM-friendly and not focusing on React-specifically. Like diffHTML could benefit from these changes as well, but allow it's VTree representation of a DOM Node to pass through HyperList and get the right positional values associated with it.